src/select.coffee

```diff
@@ -409,9 +415,9 @@ Ember.AddeparMixins.ResizeHandlerMixin, Ember.Widgets.KeyboardHelper,
       value = content.objectAt index
     else # setter
       index = content.indexOf value
-      @setHighlightedIndex index, yes
+      @setHighlightedIndex index, @get('shouldEnsureVisible')
```

Comment:
So is this a "hack" that when there are duplicated options, none of them is highlighted? I feel like we should have different way to change the way we set highlight, on an object instead of a value, which requires a different requirement for data passed in the component.